### PR TITLE
Update Ohio.md

### PR DESCRIPTION
**Pull request by alexcotarobles**
Added incidents from Canton, Dayton, and Toledo.

### CODE REVIEW

File: reports/Ohio.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## Canton
+
+### Police spray mace, fire tear gas and flash bangers at protestors, even after they have dispersed | May 29th
```
Comment (ed42311) [inline reports/Ohio.md:3]: Huh hadn't heard flash bangers as a way to describe those...

File: reports/Ohio.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+Police warn protestors blocking road that chemical agents will be deployed. The crowd mostly disperses after the warning, but police start using mace anyway. Police continue to use tear gas, flash bangers, and mace, even after protestors are out of the roadway. In part 2 of the video at 7:58, one of the used flash bang canisters is shown. It is labeled 5440 Launchable Flameless Expulsion OC.
+
+id: oh-canton-1
+
+**Links**
+* [Part 1, starts around 37:00](https://www.facebook.com/dustin.j.teeters/videos/10157947506156001/)
```
Comment (ed42311) [inline reports/Ohio.md:10]: plus one for time marker

File: reports/Ohio.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## Canton
+
+### Police spray mace, fire tear gas and flash bangers at protestors, even after they have dispersed | May 29th
+
+Police warn protestors blocking road that chemical agents will be deployed. The crowd mostly disperses after the warning, but police start using mace anyway. Police continue to use tear gas, flash bangers, and mace, even after protestors are out of the roadway. In part 2 of the video at 7:58, one of the used flash bang canisters is shown. It is labeled 5440 Launchable Flameless Expulsion OC.
+
+id: oh-canton-1
```
Comment (amardeshbd) [inline reports/Ohio.md:7]: All the record ids are same `oh-canton-1`, should they be different for each of the data set?

File: reports/Ohio.md (hunk 1 above)
Comment (emarcey) [reply reports/Ohio.md:16]: Yeah I think this should change to flashbang 😬

File: reports/Ohio.md [hunk 4]
```diff
@@ -4,11 +4,28 @@
...
 **Links**
 * [tear gas used around 9:30](https://www.facebook.com/daytondailynews/videos/203371377358112)
 
+## Canton
+
+### Police spray mace, fire tear gas and flash bangers at protestors, even after they have dispersed | May 29th
+
+Police warn protestors blocking road that chemical agents will be deployed. The crowd mostly disperses after the warning, but police start using mace anyway. Police continue to use tear gas, flash bangers, and mace, even after protestors are out of the roadway. In part 2 of the video at 7:58, one of the used flash bang canisters is shown. It is labeled 5440 Launchable Flameless Expulsion OC.
+
+tags: pepper-spray, flash-bangs, tear-gas
```
Comment (emarcey) [inline reports/Ohio.md:20]: I believe flashbangs is one word

File: reports/Ohio.md (hunk 3 above)
Comment (emarcey) [reply reports/Ohio.md:22]: Yeah these are okay because they're all the first record in each city, I believe

**Pull request merged**